cleanlab_studio/cli/dataset/upload_helpers.py

```diff
@@ -683,12 +652,16 @@ def process_dataset(
 
 
 def get_image_dataset_size(
```

Comment:
I was thinking for URL columns, including the size of the images isn't really representative of the size of the dataset since we don't have to store the images for the user. Do we also rely on the dataset size calculated here to determine whether the dataset is under size limits for training?

Reply:
So while you're right that we don't have to store the images for the user, we do care about the size of these images anyways for a few reasons:
- We use the dataset size to set a time limit for our training containers. If we don't properly account for the media size here, we won't set our training time limit correctly.
- Our required training time increases, which in turn will increase the amount we have to spend in compute. We want to properly account for this.

(A small note: I almost wrote that we incur data transfer costs, but that is not the case. AWS does not charge for in-bound data transfer, only outbound. FYI!)

There is maybe a case for handling dataset size differently for URLs, but I'm not sure that it's worth the effort. @anishathalye, do you have thoughts on this?